flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/ResultSubpartition.java

```diff
@@ -41,12 +41,19 @@
 	/** The total number of bytes (both data and event buffers) */
 	private long totalNumberOfBytes;
 
+	/** The number of non-event buffers currently in this subpartition */
+	protected int backlog;
```

Comment:
rename `backlog` to `buffersInBacklog`/`accumulatedBuffers`/`backloggedBuffers`?